libs/ui/packages/react/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -5,7 +5,7 @@ import CheckAloneRegular from "@ledgerhq/icons-ui/react/CheckAloneRegular";
 import { renderToStaticMarkup } from "react-dom/server";
 
 const Icon = () =>
-  React.cloneElement(CheckAloneRegular({ size: 13, color: "white" }), {
+  React.cloneElement(CheckAloneRegular({ size: 13, color: "currentColor" }), {
```

Comment:
Check the checkbox @sarneijim

Reply:
Now we are using lld checkbox which works correctly